src/gluonts/nursery/autogluon_tabular/estimator.py

```diff
@@ -80,6 +87,8 @@ def __init__(
         batch_size: Optional[int] = 32,
         disable_auto_regression: bool = False,
         last_k_for_val: Optional[int] = None,
+        quantile=False,
+        quantiles_topredict=[0.1, 0.5, 0.9],
```

Comment:
Would it make sense to control the behaviour with a single argument here? For example, `quantiles_to_predict=None` would give the previous behaviour, while setting it to a list of quantiles would enable the new behaviour

Reply:
make sense to me! Thanks. I'll update in the next version.